### PR TITLE
FLS-795: Basic auth added

### PR DESCRIPTION
**Pull request by hamzabinkhalid**
_Add ticket reference to Pull Request title: e.g. 'FS-123: Add content', if there is no ticket prefix with BAU_

### Change description
_A brief description of the pull request_

- [ ] Unit tests and other appropriate tests added or updated
- [ ] README and other documentation has been updated / added (if needed)
- [ ] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
_If manual testing is needed, give suggested testing steps_

### CODE REVIEW

**Comment by samuelhwilliams** [comment]
Are we only adding to the dev env, not to all non-prod envs?

**Comment by hamzabinkhalid** [comment]
> Are we only adding to the dev env, not to all non-prod envs?

Yes, Currently trying to test all services work on dev with basic auth, if nothing breaks will proceed with changes on test and uat

**Review by samuelhwilliams — DISMISSED** [review]

File: copilot/fsd-authenticator/manifest.yml [hunk 1]
```diff
@@ -120,6 +120,10 @@ environments:
   uat:
     http:
       alias: "authenticator.${COPILOT_ENVIRONMENT_NAME}.access-funding.test.levellingup.gov.uk"
+      target_container: nginx
+      healthcheck:
+        path: /healthcheck
+        port: 8080
```
Comment (samuelhwilliams) [inline copilot/fsd-authenticator/manifest.yml:126]: should this change exist for dev and test too?

Comment (hamzabinkhalid) [reply copilot/fsd-authenticator/manifest.yml:126]: it is optional but recommended.

**Review by samuelhwilliams — APPROVED** [review]

**Pull request merged**